File: platform/bb/RubyVM/src/com/rho/RhoEmptyLogger.java
package com.rho;

public class RhoEmptyLogger extends RhoLogger {
	
	public void ASSERT(boolean exp, String message) {
	}

	public void ERROR(String message, Throwable e) {
	}

	public void ERROR(String message) {
	}

	public void ERROR(Throwable e) {
	}

	public void FATAL(String message, Throwable e) {
		processFatalError();
	}

	public void FATAL(String message) {
		processFatalError();
	}

	public void FATAL(Throwable e) {
		processFatalError();
	}

	public void INFO(String message) {
	}

	public void TRACE(String message) {
	}

	public void WARNING(String message) {
	}

	public RhoEmptyLogger(){
		super("");
	}

}

File: platform/wp7/RhoRubyLib/logging/RhoEmptyLogger.cs
using System;
using rho.logging;

namespace rho.common
{
    public class RhoEmptyLogger : RhoLogger
    {
        protected override void logMessage(int severity, String msg, Exception e, boolean bOutputOnly)
        {
        }

        public override void FATAL(String message, Exception e)
        {
		    processFatalError();
	    }

	    public override void FATAL(String message) {
		    processFatalError();
	    }

        public override void FATAL(Exception e)
        {
		    processFatalError();
	    }

	    public RhoEmptyLogger() : base(""){
	
	    }
    }
}
